components/map.jsx

```diff
@@ -9,60 +9,44 @@ var Map = React.createClass({
   },
   componentDidMount: function() {
     require('mapbox.js'); // this will automatically attach to Window object.
-    this.map = L.mapbox.map(this.getDOMNode(), this.props.mapId, {
-      accessToken: this.props.accessToken
-    }).setView([43.597, -79.6139], 12);
+    require('leaflet.markercluster');
 
-    var MapLayer = L.mapbox.featureLayer(this.mapId, {accessToken: this.props.accessToken}).addTo(this.map);
-    var geoJson = [{
-      // this feature is in the GeoJSON format: see geojson.org
-      // for the full specification
-      type: 'Feature',
-      geometry: {
-        type: 'Point',
-        // coordinates here are in longitude, latitude order because
-        // x, y is the standard for GeoJSON and many formats
-        coordinates: [-79.6139,
-          43.597
-        ]
-      },
-      properties: {
-        title: 'First Point',
-        description: 'Party description?',
-        "icon": {
-          "iconUrl": "/img/map-marker.svg",
-          "iconSize": [33, 33], // size of the icon
-          "iconAnchor": [15, 15] // point of the icon which will correspond to marker's location
-        }
+    L.mapbox.accessToken = this.props.accessToken;
```

Comment:
o snap it's the return of `L.mapbox.accessToken`! Does the cluster plugin not allow an access token to be passed into it?

If we're using a global access token, then I think it shouldn't be passed-in to a `Map` component, since we don't want different `Map` components potentially stepping over each other (or conversely, requiring devs to constantly pepper the `accessToken` throughout multiple maps)... I recommend just defining a module-level variable in `map.jsx` called `accessToken`, rather than passing it in through a prop. What do you think?